tests/unit/test_index.py

```diff
@@ -84,6 +84,9 @@ def test_ext_fragment(self):
     def test_ext_query(self):
         assert '.whl' == Link('http://yo/wheel.whl?a=b').ext
 
+    def test_is_wheel(self):
+        assert Link('http://yo/wheel.whl').is_wheel
```

Comment:
I think the current style is fine, all we really need to test is the a truthy value is being returned.